app/lib/return-periods.lib.js

```diff
@@ -270,6 +270,82 @@ function _newYearElapsedQuarterFourDueDate(determinationDate, period) {
   }
 }
 
+/**
+ * Determines the current return period based on a given determination date. It calculates the return periods,
+ * maps them, sorts them by due date, and returns the current return period.
+ *
+ * @param {Date} [determinationDate=new Date()] - The date to determine the current return period from. Defaults to the current date if not provided.
+ *
+ * @returns {object} The current return period, which includes properties like `dueDate`, `startDate`, and `endDate`.
+ */
+function determineCurrentReturnPeriod(determinationDate = new Date()) {
+  const returnPeriods = determineReturnsPeriods(determinationDate)
+  const mappedReturnPeriods = _mapReturnsPeriods(returnPeriods)
+  const [currentReturnPeriod] = _sortByDueDate(mappedReturnPeriods)
+  return currentReturnPeriod
+}
+
+/**
+ * Determines the next return period based on a given determination date. It calculates the return periods,
+ * maps them, sorts them by due date, and returns the next return period.
+ *
+ * @param {Date} [determinationDate=new Date()] - The date to determine the next return period from. Defaults to the current date if not provided.
+ *
+ * @returns {object} The next return period after the given determination date, which includes properties like `dueDate`, `startDate`, and `endDate`.
+ */
+function determineNextReturnPeriod(determinationDate = new Date()) {
```

Comment:
why do you need two functions? can't you just have one function that returns both the current and the next and then do what you need with it?

Reply:
I was trying to make it more explicit. 

If we wanted to get the previous return period then it would follow this pattern. 

A single function, i think, should return all the sorted periods. But we do not need them all.

Reply:
Maybe there is a case for a single function that returns them all and it's up to the caller to use the data for it's own use case.

Reply:
`determineUpcomingReturnPeriods` ?